### PR TITLE
Adding VRF-lite

### PR DESCRIPTION
**Pull request by HenocKA**
With the blessing of @nicearma, i commit his code on VRF-lite. 
[NiceArma commit](https://github.com/nicearma/librenms/commit/97ba47cfa679513c91bef93673da9e71f2ce2a9f) : 

Add VRF-Lite logic and adapt all VRF-Lite discovery and polling modules that depend of context like:
* Discovery: arp-table
* Diccovery: bgp-peers
* Discovery: ipv4-add..
* Discovery: ipv6-add

Fix logic in : 
* Poller: bgp-peer
* Poller: ospf

Add two Mibs (CISCO-BRIDGE-DOMAIN-MIB, CISCO-CONTEXT-MAPPING-MIB)

And other little change

### CODE REVIEW

File: .gitconfig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[user]
```
Comment (SaaldjorMike) [inline .gitconfig]: This file should not be part of the PR.

Comment (HenocKA) [inline .gitconfig]: Right, it's a mistake.

**Comment by f0o** [comment]
PR needs a rebase

**Comment by HenocKA** [comment]
Sorry, I closed this pull request because of errors.
